### ISSUE TITLE
`rotvec_to_rotmat` function detach the tensor's gradient

### ISSUE THREAD

**Issue by zxiaomzxm** [naver/roma#1]
Hi there, I use the rotvec_to_rotmat fucntion, and I found that this function detach the gradient flow. Is there any way to keep gradients flow in this function?
```python
tvec = torch.randn(1, 3, requires_grad=True)
rot = roma.rotvec_to_rotmat(rotvec)
print(rot.requires_grad) # get False
```

**Comment by zxiaomzxm** [comment]
fixed...

**Comment by rbregier** [comment]
Hi, thanks for using roma.
Indeed it seems that the bug was on your side:
```python
import torch, roma
tvec = torch.randn(1, 3, requires_grad=True)
rot = roma.rotvec_to_rotmat(tvec)
print(rot.requires_grad) # returns True
```
